js/web/webpack.config.js

```diff
@@ -1,90 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
 const path = require('path');
 const webpack = require('webpack');
 const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
 const minimist = require('minimist');
 
-function buildAllConfig({
-  suffix = '',
-  format = 'umd',
-  target = 'ES2017',
-  mode = 'production',
-  devtool = 'source-map'
-}) {
+// common config for release bundle
+function buildConfig({ filename, format, target, mode, devtool }) {
   return {
     entry: path.resolve(__dirname, 'lib/index.ts'),
     output: {
       path: path.resolve(__dirname, 'dist'),
-      filename: `ort${suffix}.js`,
+      filename,
       library: {
-        name: 'ort',
         type: format
       }
     },
-    externals: {
-      'fs': 'fs',
-      'path': 'path',
-      'util': 'util',
-    },
     resolve: { extensions: ['.ts', '.js'] },
     plugins: [new webpack.WatchIgnorePlugin({ paths: [/\.js$/, /\.d\.ts$/] })],
     module: {
       rules: [{
-        test: /\.tsx?$/,
+        test: /\.ts$/,
         use: [
           {
             loader: 'ts-loader',
             options: {
-              compilerOptions: { target: target }
+              compilerOptions: { target }
             }
           }
         ]
       }]
     },
-    mode: mode,
-    devtool: devtool,
+    mode,
+    devtool
   };
 }
 
-function buildWebConfig({
+// "ort{.min}.js" config
+function buildOrtConfig({
+  suffix = '',
+  target = 'es5',
+  mode = 'production',
+  devtool = 'source-map'
+}) {
+  const config = buildConfig({ filename: `ort${suffix}.js`, format: 'umd', target, mode, devtool });
+  // set global name 'ort'
+  config.output.library.name = 'ort';
+  // do not use those node builtin modules in browser
+  config.resolve.fallback = { path: false, fs: false, util: false };
+  return config;
+}
+
+// "ort-web{.min|.node}.js" config
+function buildOrtWebConfig({
   suffix = '',
   format = 'umd',
-  target = 'ES2017',
+  target = 'es5',
```

Comment:
can we just make this es6 by default? I see most of the configs below are using es6.

Reply:
they are different targets. for compatibility concerns, we use es5 by default; for es6, we add ".es6" in the file name, so they are separated files